### PR TITLE
update README with markdown

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

**Comment by shawndwells** [comment]
@spcsspcs could you review?

**Comment by spcsspcs** [comment]
Made one comment, the rest look good to me.

**Comment by shawndwells** [comment]
@spcsspcs does not appear the comment made it into the PR review. If you're OK with the change we can merge this PR to create incremental progress & address the comment in another PR.

**Comment by spcsspcs** [comment]
This is the comment I made on line 45 of the readme file:

I suggest we remove and say something like --username ${CS_CLIENT_ID}
and highlight password is also: ${CS_CLIENT_SECRET} since the username and password is really the CS API client ID and Secret

Sure, feel free to merge this and we can open up a new PR.

File: README [hunk 1]
```diff
@@ -35,32 +36,42 @@ required arguments:
   --tag TAG             docker image tag
   --cloud {us-1,us-2,eu-1}
                         CS cloud name
+``
+
+## Example Scan
+This requires your image to exist locally, e.g. run ``docker pull`` prior to executing this script.
+
+``shell
+$ python cs_scanimage.py --user <username> --repo <repo> --tag <tag> --cloud <cloud>
```
[the remark this answers is not in the document]
Comment (shawndwells) [reply README:45]: @spcsspcs opened https://github.com/CrowdStrike/cicd-falcon-tutorial/pull/3 to start converstation about updating variable names and output

For now, what do you think about merging this PR, scoped to formating the README, and use https://github.com/CrowdStrike/cicd-falcon-tutorial/pull/3 to workshop the underlying code/variables?

**Review by ashwinathalye-cs — APPROVED** [review]

**Pull request merged**